sdk/pom.xml

```diff
@@ -174,9 +174,6 @@
           <excludePackageNames>pubsub:greengrass:identity:jobs:pubsubstress:rawpubsub:pkcs11pubsub:windowscertpubsub:shadow</excludePackageNames>
```

Comment:
if just refers to the sample that says it's package is literally just [package greengrass;](https://github.com/aws/aws-iot-device-sdk-java-v2/blob/c04ce42a7c83bb6aa5f8e51b2cd1af44f99fc7d1/samples/Greengrass/src/main/java/greengrass/BasicDiscovery.java#L6)

It doesn't `exclude software.amazon.awssdk.aws.greengrass`, that package is still showing up in the docs

Reply:
nevermind. no more includes list, no more excludes list

we weren't at risk of publishing sample docs anyway, because we do this `cp -r sdk/target/site/apidocs/ docs/`

but I also tweaked make-docs.sh so it only even tries to do javadoc for the sdk